app/src/main/webapp/app/error/error.ctrl.js

```diff
@@ -137,9 +137,44 @@ esbMessageAdminApp.controller('ErrorsSearchCtrl', [
     $scope.$watch('messageSelections',
       function() {
         if ($scope.messageSelections.length > 0) {
-          $rootScope.selectedMessage = $scope.messageSelections[0];
+          $scope.selectedMessage = $scope.messageSelections[0];
+          EsbMessageService.getMessage($scope.selectedMessage.id).then(
```

Comment:
Great! For clarity / sanity, I would probably avoid assigning "selectedMessage" to the scope at all here, since it's never used by a template, and _actually_ the real "selected message" is defined by `$scope.message`. So for clarity I would use a local variable instead, and maybe name it something like, `var firstInSelection = $scope.messageSelections[0];`

Otherwise, reading it I would assume `selectedMessage` has greater significance since it's assigned to scope, and then I would wonder, "what the heck is the difference between that and just `message`?"